{{cookiecutter.github_repository}}/settings/common.py

```diff
@@ -25,27 +25,44 @@
     "django.contrib.messages",
     "django.contrib.staticfiles",
     "django.contrib.admin",
-    # "django.contrib.humanize",  # Useful template tags
-    "{{ cookiecutter.main_module }}.base",
-    "{{ cookiecutter.main_module }}.users",
+]
+
+THIRD_PARTY_APPS = [
+    "django_filters",
```

Comment:
Please correct me if I'm wrong, but I think `django_filters` is used by graphene relay, so we can put it inside the if block.